src/plugins/intel_cpu/src/nodes/paged_attn.cpp

```diff
@@ -272,6 +272,14 @@ bool PagedAttention::isSupportedOperation(const std::shared_ptr<const ov::Node>&
 
 ov::element::Type PagedAttention::getRuntimePrecision() const {
     auto rtPrecision = getOriginalInputPrecisionAtPort(0);
+#if defined(OPENVINO_ARCH_ARM64)
+    if (rtPrecision == ov::element::f16) {
+        rtPrecision = ov::element::f16;
+    } else {
+        rtPrecision = ov::element::f32;
+    }
+    return rtPrecision;
+#endif
```

Comment:
Shouldn't we now check for the cache type to select the proper runtime precision? `fp16` supports only `u8` KV cache, but it is not guarantied that the cache has this precision, therefore if it has a precision other than `u8`, the implementation must switch to `fp32`.

Reply:
Discussed offline. This change doesn't alter the existing master behavior negatively.